src/coreclr/vm/loaderallocator.hpp

```diff
@@ -332,7 +332,7 @@ class LoaderAllocator
     bool                m_fTerminated;
     bool                m_fMarked;
     int                 m_nGCCount;
-    bool                m_IsCollectible;
+    BYTE                m_IsCollectible;
```

Comment:
I couldn't make this `const BYTE` because that created build errors in the DAC-ize constructor created by the `VPTR_BASE_VTABLE_CLASS(LoaderAllocator)` macro